pkg/cmd/util/clientcmd/errors.go

```diff
@@ -0,0 +1,44 @@
+package clientcmd
+
+import "strings"
+
+type ClientError struct {
+	reason int
+	msg    string
+}
+
+func (e ClientError) Error() string {
+	return e.msg
+}
+
+const (
+	notConfiguredReason               = 1
+	certificateAuthorityUnknownReason = 2
+
+	certificateAuthorityUnknownMsg = "The server uses a certificate signed by unknown authority. You may need to use the --certificate-authority flag to provide the path to a certificate file for the certificate authority, or --insecure-skip-tls-verify to bypass the certificate check and use insecure connections."
+	notConfiguredMsg               = `OpenShift is not configured. You need to run the login command in order to create a default config for your server and credentials:
+  osc login
+You can also run this command again providing the path to a config file directly, either through the --config flag of the OPENSHIFTCONFIG environment variable.
+`
+)
+
+func WrapClientErrors(err error) error {
```

Comment:
If we do end up keeping this, it's not wrapping errors, it's completely replacing them, so either rename appropriately or rewrite to match

If I'm understanding intent correctly, you're simply trying to find a way to display a pretty message to users.  I would suggest having a method like: GetPrettyMessageForError(err) that simply checks a list of know kinds of error messages and gives you back a pretty string to display in those cases where you care.  That way our errors remain consistent throughout the API and code, but users still see something pretty.  It would also give you a good spot to intercept other kinds of errors.